content/en/apps/guides/data/analytics/testing-dbt-models.md

```diff
@@ -0,0 +1,138 @@
+Records that should be deleted are properly removed from both tables.
+All relevant fields between the two tables match, preserving data accuracy and integrity.
+{{< figure src="contact-sql.png" link="contact-sql.png" class=" center col-16 col-lg-12" >}}
+
+### Unit Tests
+
+Unit tests are defined in a YML file inside a `tests` folder in each `model` group folder. The input format of choice is CSV using fixtures, defined in `/test/fixture` folder.
+
+The following image shows the content in `/models/contacts/tests/contacts.yml` file showing the test to validate transformation and data integrity for `contact` model.
+{{< figure src="contact-unit-test.png" link="contact-unit-test.png" class=" center col-16 col-lg-12" >}}
+
+The following images shows the content of the input fixtures.
+{{< figure src="contact-document-metadata-initial.png" link="contact-document-metadata-initial.png" class=" center col-16 col-lg-12" >}}{{< figure src="contact-source-table-initial.png" link="contact-source-table-initial.png" class=" center col-16 col-lg-12" >}}
+
+And finally, the fixture representing the expected data:
+{{< figure src="contact-initial-expected.png" link="contact-initial-expected.png" class=" center col-16 col-lg-12" >}}
+
+## Test execution using Docker
+
+By running dbt tests in a Dockerized environment, we isolate dependencies and configurations, making the testing process more reproducible and easier to manage. These scripts and configuration files work together to automate the process of setting up a testing environment, running dbt models and tests:
+
+`docker-compose.yml`: Defines Docker services. It starts by running PostgreSQL with the initial setup using init.sql and then builds and runs a dbt Docker container configured to connect to PostgreSQL.
+
+`init.sql`: A SQL script that initializes the PostgreSQL database by creating the required schema and table.
+
+`dbt/Dockerfile`: Builds the dbt Docker image. It sets up the Python environment, installs dbt and its dependencies, and copies the necessary project files into the container.
+
+`run_dbt_tests.sh`: This script orchestrates the process of running dbt tests using Docker. It starts PostgreSQL, waits for it to initialize, runs the dbt container to execute tests, and then cleans up by stopping and removing all containers.
+
+`run_dbt_tests_docker.sh`: This script runs inside the dbt container. It sets the profiles directory, installs dbt dependencies, runs the dbt models, and ***executes the tests***.
+
+`profile.yml`: The dbt profile configuration file that defines the database connection settings, using environment variables to connect to the PostgreSQL database.
+
+### Prerequisites
+- `Docker`
+
+### Run the tests
```

Comment:
Can we add a short sentence at the end that explains quickly what the developer should see after running the scripts? Something like: Docker container starts, tests run, tests pass, Docker containers are stopped and removed.